website/docs/r/team_organization_member.html.markdown

```diff
@@ -47,9 +47,14 @@ The following arguments are supported:
 
 ## Import
 
-A team member can be imported; use `<TEAM ID>/<ORGANIZATION MEMBERSHIP ID>`
+A team member can be imported; use `<TEAM ID>/<ORGANIZATION MEMBERSHIP ID>` or `<ORGANIZATION NAME>/<USER EMAIL>/<TEAM NAME>`
```

Comment:
I'm actually having trouble importing, but I can see that the correct member/team is being located.

```
$ terraform import tfe_team_organization_member.foo hashicorp/alisdair@hashicorp.com/test-team
tfe_team_organization_member.foo: Importing from ID "hashicorp/alisdair@hashicorp.com/test-team"...
tfe_team_organization_member.foo: Import prepared!
  Prepared tfe_team_organization_member for import
tfe_team_organization_member.foo: Refreshing state... [id=team-ssoLN3CDbW4u1HdG/ou-cWsiUYr3GeMaPJKt]
╷
│ Error: Cannot import non-existent remote object
│
│ While attempting to import an existing object to "tfe_team_organization_member.foo", the provider detected that no object exists with the given id. Only
│ pre-existing objects can be imported; check that the id is correct and that it is associated with the provider's configured region or endpoint, or use "terraform
│ apply" to create a new remote object for this resource.
╵

```

Reply:
@brandonc is the user a member of the given team? I believe that is the error I see when trying to import a team membership which doesnt exist (i.e. alisdair is in the org, and the team exists, but isnt a member of the team).

I am battling some dev env demons right now, but will double check as soon as I get those sorted!

Reply:
I went to improve this error message by having the import read the [organization memberships from the team](https://developer.hashicorp.com/terraform/cloud-docs/api-docs/teams#available-related-resources) it identifies based on org/team-name, but I noticed that to do that properly I will need to [update go-tfe](https://github.com/hashicorp/go-tfe/blob/main/team.go#LL203) to accept `include` options on Team Read.

@brandonc assuming I am correct and the error you are seeing is because the user is not part of the team, and just an unhelpful (pre-existing) error message instead of a new failure...what would you think about leaving that failure in with this PR? I can update go-tfe as necessary to improve it, and circle back once that is done.